### PR TITLE
update pd-ctl doc

### PR DESCRIPTION
**Pull request by ran-huang**
Signed-off-by: Ran <huangran@pingcap.com>

### What is changed, added or deleted? (Required)

Update the pd-ctl doc.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/3280
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [x] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: pd-control.md [hunk 1]
```diff
@@ -218,231 +217,230 @@ export PD_ADDR=http://127.0.0.1:2379 &&
...
 
-```bash
->> config set split-merge-interval 24h
-```
+    ```bash
+    >> config set split-merge-interval 24h
+    ```
 
-`patrol-region-interval` 控制 replicaChecker 检查 Region 健康状态的运行频率，越短则运行越快，通常状况不需要调整。
+- `enable-one-way-merge` 用于开启是否只允许和相邻的后一个 Region 进行合并。当设置为 false 时，PD 允许与相邻的前后 Region 进行合并。
```
Comment (TomShawn) [inline pd-control.md:274]:
```suggestion
- `enable-one-way-merge` 用于控制是否只允许和相邻的后一个 Region 进行合并。当设置为 `false` 时，PD 允许与相邻的前后 Region 进行合并。
```

File: pd-control.md [hunk 2]
```diff
@@ -218,231 +217,230 @@ export PD_ADDR=http://127.0.0.1:2379 &&
...
 
-```bash
->> config set patrol-region-interval 10ms
-```
+    ```bash
+    >> config set enable-one-way-merge true
+    ```
 
-`max-store-down-time` 为 PD 认为失联 store 无法恢复的时间，当超过指定的时间没有收到 store 的心跳后，PD 会在其他节点补充副本。
+- `enable-cross-table-merge` 用于开启跨表 Region 的合并。当设置为 false 时，PD 不会合并不同表的 Region。该选项只在键类型为 "table" 时生效。
```
Comment (TomShawn) [inline pd-control.md:284]:
```suggestion
- `enable-cross-table-merge` 用于开启跨表 Region 的合并。当设置为 `false` 时，PD 不会合并不同表的 Region。该选项只在键类型为 "table" 时生效。
```

File: pd-control.md [hunk 3]
```diff
@@ -218,231 +217,230 @@ export PD_ADDR=http://127.0.0.1:2379 &&
...
->> config set max-store-down-time 30m
-```
+    ```bash
+    >> config set enable-cross-table-merge true
+    ```
 
-通过调整 `leader-schedule-limit` 可以控制同时进行 leader 调度的任务个数。
-这个值主要影响 *leader balance* 的速度，值越大调度得越快，设置为 0 则关闭调度。
-Leader 调度的开销较小，需要的时候可以适当调大。
+- `key-type` 用于指定集群的键编码类型。支持的选项有 `["table", "raw", "txn"]`，默认值为 "table"。
```
Comment (TomShawn) [inline pd-control.md:294]:
```suggestion
- `key-type` 用于指定集群的键编码类型。支持的类型有 `["table", "raw", "txn"]`，默认值为 "table"。
```

File: pd-control.md [hunk 4]
```diff
@@ -218,231 +217,230 @@ export PD_ADDR=http://127.0.0.1:2379 &&
...
-```
+    {{< copyable "" >}}
 
-`hot-region-schedule-limit` 控制同时进行的 Hot Region 调度的任务，设置为 0 则关闭调度。这个值不宜调得过大，否则可能对系统性能造成影响。
+    ```bash
+    >> config set leader-schedule-limit 4
+    ```
 
-最多同时进行 4 个 Hot Region 调度：
+- 通过调整 `region-schedule-limit` 可以控制同时进行 Region 调度的任务个数。这个值可以避免创建过多的 Region balance 算子。默认值为 `2048`，对所有大小的集群都足够。设置为 `0` 则关闭调度。Region 调度的速度通常受到 `store-limit` 的限制，但除非你熟悉相关设置，否则不推荐自定义该参数。
```
Comment (TomShawn) [inline pd-control.md:339]: 这里的 "operator" 可能不是"算子"。
@rleungx PTAL
```suggestion
- 通过调整 `region-schedule-limit` 可以控制同时进行 Region 调度的任务个数。这个值可以避免创建过多的 Region balance operator。默认值为 `2048`，对所有大小的集群都足够。设置为 `0` 则关闭调度。Region 调度的速度通常受到 `store-limit` 的限制，但除非你熟悉该设置，否则不推荐自定义该参数。
```

File: pd-control.md [hunk 5]
```diff
@@ -218,231 +217,230 @@ export PD_ADDR=http://127.0.0.1:2379 &&
...
-`enable-placement-rules` 用于开启 placement rules。
+- `enable-remove-extra-replica` 用于开启删除多余副本的特性。当设置为 false 时，PD 不会为副本数过多的 Region 删除多余副本。
+
+- `enable-location-replacement` 用于开启隔离级别检查。当设置为 false 时，PD 不会通过调度来提升 Region 副本的隔离级别。
+
+- `enable-debug-metrics` 用于开启 debug 的 metrics。当设置为 true 时，PD 会开启一些 metrics，比如 `balance-tolerant-size` 等。
+
+- `enable-placement-rules` 用于开启 placement rules。
+
+- `store-limit-mode` 用于控制 store 限速机制的模式。主要有两种模式：`auto` 和 `manual`。`auto` 模式下会根据 load 自动进行调整（实验性功能）。
```
Comment (TomShawn) [inline pd-control.md:439]:
```suggestion
- `store-limit-mode` 用于控制 store 限速机制的模式。主要有两种模式：`auto` 和 `manual`。`auto` 模式下会根据 load 自动进行平衡调整（实验性功能）。
```

File: pd-control.md [hunk 6]
```diff
@@ -958,6 +957,65 @@ Encoding 格式示例：
...
+    >> scheduler config balance-hot-region-scheduler set min-hot-byte-rate 100
+    ```
+
+- `min-hot-key-rate` 指计数的最小 key，通常为 10。
+
+    ```bash
+    >> scheduler config balance-hot-region-scheduler set min-hot-key-rate 10
+    ```
+
+- `max-zombie-rounds` 指一个算子可以被认为是 pending influence 的最大心跳次数。如果将它设置为更大的值，更多的算子可能会被包括在这个范围内。通常不需要修改这个值。pending influence 指的是在调度过程中产生的但仍有影响的算子影响。
```
Comment (TomShawn) [inline pd-control.md:995]: @lhy1024 PTAL
```suggestion
- `max-zombie-rounds` 指一个 operator 可被纳入 pending influence 所允许的最大心跳次数。如果将它设置为更大的值，更多的 operator 可能会被纳入 pending influence。通常用户不需要修改这个值。pending influence 指的是在调度中产生的、但仍生效的影响。
```

File: pd-control.md [hunk 7]
```diff
@@ -958,6 +957,65 @@ Encoding 格式示例：
...
+    >> scheduler config balance-hot-region-scheduler set max-zombie-rounds 3
+    ```
+
+- `max-peer-number` 指最多要被解决的 peer 数量。这个配置可避免调度器处理速度过慢。
+
+    ```bash
+    >> scheduler config balance-hot-region-scheduler set max-peer-number 1000
+    ```
+
+- `byte-rate-rank-step-ratio`，`key-rate-rank-step-ratio`，和 `count-rank-step-ratio` 分别控制 byte、key、count 的 step ranks。rank step ratio 决定了计算 rank 时的 step 值。`great-dec-ratio` 和 `minor-dec-ratio` 控制 `dec` 的 rank。统舱不需要修改这些配置项。
```
Comment (TomShawn) [inline pd-control.md:1007]:
```suggestion
- `byte-rate-rank-step-ratio`、`key-rate-rank-step-ratio` 和 `count-rank-step-ratio` 分别控制 byte、key、count 的 step ranks。rank step ratio 决定了计算 rank 时的 step 值。`great-dec-ratio` 和 `minor-dec-ratio` 控制 `dec` 的 rank。通常用户不需要修改这些配置项。
```

File: pd-control.md [hunk 8]
```diff
@@ -1021,16 +1079,16 @@ Encoding 格式示例：
...
+>> store limit                         // 显示所有 store 添加和删除 peer 的速度上限
+>> store limit add-peer                // 显示所有 store 添加 peer 的速度上限
+>> store limit remove-peer             // 显示所有 store 删除 peer 的速度上限
+>> store limit all 5                   // 设置所有 store 添加和删除 peer 的速度上限为每分钟 5 个
+>> store limit 1 5                     // 设置 store 1 添加和删除 peer 的速度上限为每分钟 5 个
+>> store limit all 5 add-peer          // 设置所有 store 添加 peer 的速度上限为每分钟 5 个
+>> store limit 1 5 add-peer            // 设置 store 1 添加 peer 的速度上限为每分钟 5 个
+>> store limit 1 5 remove-peer         // 设置 store 1 删除 peer 的速度上限为每分钟 5 个
+>> store limit all 5 remove-peer       // 设置所有 store 删除 peer 的速度上限为每分钟 5 个
+>> store limit-scene                   // 显示所有 limit 场景（实验性功能）
```
Comment (TomShawn) [inline pd-control.md:1091]:
```suggestion
>> store limit-scene                   // 显示所有的限速场景（实验性功能）
```

**Comment by TomShawn** [comment]
These updates in the original PR are missing in this PR:

+ https://github.com/pingcap/docs/pull/3280/files#diff-71fa457167c89bb4e3cb651dc0484326R476-R492
+ https://github.com/pingcap/docs/pull/3280/files#diff-71fa457167c89bb4e3cb651dc0484326R645

**Comment by ran-huang** [comment]
@TomShawn Updated the 2 missing parts: c2ce5d5. PTAL, thanks!

**Comment by TomShawn** [comment]
Rest LGTM except for https://github.com/pingcap/docs-cn/pull/4276#discussion_r471218648 and https://github.com/pingcap/docs-cn/pull/4276#discussion_r471214319.

File: pd-control.md (hunk 4 above)
Comment (rleungx) [reply pd-control.md:339]: 能保留 operator 这个概念吗？翻译成中文的话感觉会有点奇怪

**Comment by ran-huang** [comment]
@rleungx 保留了 "operator". PTAL, thanks!

**Pull request merged**